bosh/releases/pre_render_scripts/eirini/eirini-persi-broker/bpm/patch_bpm.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+target="/var/vcap/all-releases/jobs-src/eirini/eirini-persi-broker/templates/bpm.yml.erb"
+
+# Patch BPM, since we're actually running in-cluster without BPM
+PATCH=$(cat <<'EOT'
+@@ -3,17 +3,4 @@ processes:
+     executable: /var/vcap/packages/eirini-persi-broker/bin/eirini-persi-broker
+     args: []
+     env:
+-      KUBERNETES_SERVICE_HOST: "<%= p("eirini-persi-broker.kube_service_host") %>"
+-      KUBERNETES_SERVICE_PORT: "<%= p("eirini-persi-broker.kube_service_port") %>"
+       BROKER_CONFIG_PATH: /var/vcap/jobs/eirini-persi-broker/config/eirini-persi-broker.yml
+-    <% if properties.opi&.k8s&.host_url.nil? %>
+-    # The ServiceAccount admission controller has to be enabled.
+-    # https://kubernetes.io/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod
+-    additional_volumes:
+-    - path: /var/run/secrets/kubernetes.io/serviceaccount/token
+-      mount_only: true
+-    - path: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+-      mount_only: true
+-    - path: /var/run/secrets/kubernetes.io/serviceaccount/namespace
+-      mount_only: true
+-    <% end %>
+EOT
+)
+
+# Only patch once
```

Comment:
You might want to adapt this according to https://github.com/SUSE/kubecf/pull/261 in order to keep things consistent.

Reply:
Fixed here: https://github.com/SUSE/kubecf/pull/249/commits/22110fc4f3bd5fa57b25f966d993742e41040527